packages/sdk/client-services/src/packlets/invitations/invitations-service.ts

```diff
@@ -43,6 +48,7 @@ export class InvitationsServiceImpl implements InvitationsService {
   createInvitation(options: Invitation): Stream<Invitation> {
     let invitation: CancellableInvitation;
 
+    const savePersistentInvitationCtx = new Context();
```

Comment:
unhandled error here (from `addPersistentInvitaion`).

I'd move the whole `scheduleTaskCall` into `return new Stream<Invitation>(({ next, close }) => {` so that you can bubble errors up